j2lint/linter/collection.py

```diff
@@ -41,6 +41,7 @@ def run(self, file_dict):
         """
         text = ""
         errors = []
+        lint_warnings = []
```

Comment:
Variable should be named `warnings` instead of `lint_warnings` to keep it consistent with the name errors

Reply:
corrected it.